### PR TITLE
adding vip id and subnet id to lbpoolv2 response

### PR DESCRIPTION
**Pull request by drmaas**

### CODE REVIEW

**Comment by drmaas** [comment]
@emjburns @gondor ^^^

**Comment by drmaas** [comment]
see http://developer.openstack.org/api-ref/networking/v2-ext/index.html?expanded=show-pool-details-detail

File: core/src/main/java/org/openstack4j/model/network/ext/LoadBalancerV2.java [hunk 1]
```diff
@@ -34,14 +34,14 @@
...
      */
-    String getVIPSubnetId();
+    String getVipSubnetId();
 
     /**
-     * @return The VIP address of the loadbalancer.
+     * @return The vip address of the loadbalancer.
      */
-    String getVIPAddress();
+    String getVipAddress();
```
Comment (emjburns) [inline core/src/main/java/org/openstack4j/model/network/ext/LoadBalancerV2.java]: Good call. Looks like we should also add vip_port_id? my return json looks like 
```{
    "loadbalancer": {
        "description": "",
        "admin_state_up": true,
        "tenant_id": "1fd5a8034d7d4359b0ec42b0cec56b76",
        "provisioning_status": "ACTIVE",
        "listeners": [
            {
                "id": "996d7a0a-4de0-4ae4-ba82-e2f05f5a4ded"
            }
        ],
        "vip_address": "10.0.0.3",
        "vip_port_id": "7cd5d967-598b-47c3-9cf1-a01525f9b951",
        "provider": "octavia",
        "vip_subnet_id": "c0bbd43a-5246-49f4-9d21-29bde5bd333d",
        "id": "33494c48-30dc-4571-8644-f7b70f6aabd0",
        "operating_status": "ONLINE",
        "name": "DemoLB0"
    }
}```

File: core/src/main/java/org/openstack4j/openstack/networking/domain/ext/NeutronLbPoolV2.java [hunk 2]
```diff
@@ -148,6 +154,16 @@ public String getHealthMonitorId(){
     }
 
     @Override
+    public String getSubnetId() {
```
Comment (emjburns) [inline core/src/main/java/org/openstack4j/openstack/networking/domain/ext/NeutronLbPoolV2.java]: These objects do not appear in my LbPoolV2 Json: 
```{"pool": {"lb_algorithm": "ROUND_ROBIN", "protocol": "HTTP", "description": "this is a lb pool", "admin_state_up": true, "tenant_id": "1fd5a8034d7d4359b0ec42b0cec56b76", "session_persistence": null, "healthmonitor_id": "d7caa104-d481-4e03-9c80-a7cfea2f1f2b", "listeners": [{"id": "996d7a0a-4de0-4ae4-ba82-e2f05f5a4ded"}], "members": [{"id": "c788f060-9537-4294-b090-187dc3c5527d"}], "id": "12cfb1fd-aff9-49e4-ae94-96cdbeb7256a", "name": "DemoPool0"}}```

File: core/src/main/java/org/openstack4j/model/network/ext/LbPoolV2.java [hunk 3]
```diff
@@ -71,4 +71,14 @@
      * @return Health monitor associated with the pool.
      */
     String getHealthMonitorId();
+
```
Comment (emjburns) [inline core/src/main/java/org/openstack4j/model/network/ext/LbPoolV2.java]: Same comment as below - these dont appear in my lbpoolv2 json.

File: core/src/main/java/org/openstack4j/openstack/networking/domain/ext/NeutronLbPoolV2.java (hunk 2 above)
Comment (emjburns) [inline core/src/main/java/org/openstack4j/openstack/networking/domain/ext/NeutronLbPoolV2.java]: Ok cool. Must not be configured on my openstack!

**Comment by emjburns** [comment]
LGTM

File: core/src/main/java/org/openstack4j/openstack/networking/domain/ext/NeutronLbPoolV2.java (hunk 2 above)
Comment (drmaas) [inline core/src/main/java/org/openstack4j/openstack/networking/domain/ext/NeutronLbPoolV2.java]: If subnetId and vipId are not actually in the response, then I can revert those. Let me know after you do some testing.

Comment (emjburns) [inline core/src/main/java/org/openstack4j/openstack/networking/domain/ext/NeutronLbPoolV2.java]: So if you look at this documentation a lot of it was just copy-pasted from the old stuff. check out http://developer.openstack.org/api-ref/networking/v2-ext/index.html?expanded=show-pool-details-detail,id370-detail ( /v2.0/lb/pools/{pool_id}) its exactly the same text. And the pool object has changed. You can see that the new object in this documentation doesn't even contain the 'listener' field. I would refrain from using this documentation. Instead, use https://wiki.openstack.org/wiki/Neutron/LBaaS/API_2.0 (although even this isn't perfect)

File: core/src/main/java/org/openstack4j/model/network/ext/LoadBalancerV2.java (hunk 1 above)
Comment (emjburns) [inline core/src/main/java/org/openstack4j/model/network/ext/LoadBalancerV2.java]: The documentation does not show vip_port_id (https://wiki.openstack.org/wiki/Neutron/LBaaS/API_2.0#Retrieve_a_specific_Load_Balancer) but i'm seeing it when I test. json above

**Pull request merged**